docs/src/examples/components/Tooltip/Usage/TooltipExampleAsync.shorthand.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Button, Tooltip, Segment } from '@stardust-ui/react'
+
+class AsyncDataLoader extends React.Component<any, any> {
+  state = {
+    data: 'loading..',
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({
+        data: <Segment styles={{ minHeight: '300px' }}>Hello from loaded data!</Segment>,
+      })
+      this.props.onLoaded()
+    }, 1000)
+  }
+
+  render() {
+    return this.state.data
+  }
+}
+
+const TooltipExampleAsync = () => (
+  <Tooltip
+    trigger={<Button icon="expand" content="Hover me!" />}
+    renderContent={updatePosition => ({ content: <AsyncDataLoader onLoaded={updatePosition} /> })}
+  />
+)
+
+export default TooltipExampleAsync
```

Comment:
Is it required for initial implementation, if so can we reuse `positionDependencies` prop and omit this `renderContent` stuff?

Reply:
+1 for `positionDependencies` or omitting this for now as it's gonna need reworking (see #1366)

Reply:
Ok, will revert this changes